src/routes/add-article.ts

```diff
@@ -30,8 +30,8 @@ export default (): AppMiddleware => (
       throw new createHttpError.BadRequest(`Article must have a blank node identifier (${id.value} given)`);
     }
 
-    if (request.dataset.match(id, schema.name).size === 0) {
-      throw new createHttpError.BadRequest(`Article must have at least one ${schema.name.value}`);
+    if (request.dataset.match(id, schema('name')).size === 0) {
+      throw new createHttpError.BadRequest(`Article must have at least one ${schema('name').value}`);
```

Comment:
This had to be reverted due to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41082#discussion_r360391625.